packages/loaders/src/styled/StyledLoadingPlaceholder.ts

```diff
@@ -14,10 +14,10 @@ export const StyledLoadingPlaceholder = styled.div.attrs({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
   role: 'progressbar'
-})<{ fontSize: string | number }>`
-  display: inline;
-  width: 1em;
-  height: 0.9em;
+})<{ fontSize: string | number; width?: string; height?: string }>`
+  display: block;
```

Comment:
If I'm following, the best we can do here is `inline-block`. Current PR's `block` display would introduce unexpected layout issues.

Reply:
Fixed in https://github.com/zendeskgarden/react-components/pull/1233/commits/895ee404126283a828e8af47c4c615101dd36f85